src/components/sbb-header/sbb-header.scss

```diff
@@ -39,26 +33,39 @@
 }
 
 :host([hide-on-scroll][data-fixed][data-visible]) {
-  --sbb-header-inset-block-start: 0;
-
   // Show transition
   --sbb-header-transition-timing: cubic-bezier(0, 0, 0.2, 1);
 }
 
 .sbb-header {
   position: var(--sbb-header-position);
   inset: 0 0 auto;
-  inset-block-start: var(--sbb-header-inset-block-start);
   background: var(--sbb-color-white-default);
   z-index: 10;
-  transition-property: var(--sbb-header-transition-property);
-  transition-duration: var(--sbb-header-transition-duration);
-  transition-timing-function: var(--sbb-header-transition-timing);
+  transition: {
+    property: var(--sbb-header-transition-property);
```

Comment:
I think we should remove the var(--sbb-header-transition-property) and inline it